tests/UnitTests/CoreGraphics.drawing/EntryPoint.cpp

```diff
@@ -54,7 +48,22 @@ class CommandLineDrawingTestConfigImpl : public DrawingTestConfigImpl {
 
 public:
     CommandLineDrawingTestConfigImpl(int argc, char** argv)
```

Comment:
I agree in principle. I'll let you handle that. There were already a number of ifdefs in there for WIN32 so it seems like we need to think more about how to divide up the entrypoint code for the two.